website/docs/provider_versions.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "google"
+page_title: "Google Provider Versions"
+sidebar_current: "docs-google-provider-versions"
+description: |-
+  How to use the different Google Provider versions
+---
+
+# Google Provider Versions
+
+The Google Provider comes in two versions:
+
+* terraform-provider-google
+* terraform-provider-google-beta
+
+All GA products and features will be available in both versions of the provider.
```

Comment:
nit: let's say "are"? It keeps the tense the same, and doesn't promise things for the future.